### ISSUE TITLE
Failed to resolve: com.sagar:coroutinespermission:2.0.3

### ISSUE THREAD

**Issue by D10NGYANG** [sagar-viradiya/eazypermissions#50]

**Comment by D10NGYANG** [comment]
I re-added jcenter() to the repositories and it worked.
But android studio show tips "this repository is going to shut down soon."

**Comment by sagar-viradiya** [comment]
@D10NGYANG Yes jcenter will be shutting down next year and I will be working on moving this library to maven.

**Comment by esdudnik** [comment]
@sagar-viradiya Any chances that library will be reuploaded to maven central? Thanks
